### DISCUSSION TITLE
Why name Avalonia? (Having a hard time even remembering it)

### DISCUSSION THREAD

**Discussion by dinbtechit** [AvaloniaUI/Avalonia#13882] [Ideas]
Interesting framework is following Flutter's footsteps, and it looks like it is going to make it easy for .NET C# developers to onboard. However, why make the framework name so complicated? Do not roll off the tongue. IMO.

**Comment by timunie** [comment]
Do you prefer Perspex? 

More about it here: https://avaloniaui.net/Blog/10-years-of-avalonia

**Reply by dinbtechit** [reply]
Wow interesting article.  Thanks for sending that.

![image](https://github.com/AvaloniaUI/Avalonia/assets/17984781/8b345518-008e-4403-b640-e1334e4d0ae7)

Some of the simpler names: (Something that's more easy to relate to across the global and not fixed to a specific region or one demographic)

1. Glide Mobile Native - Represents the smooth transition for .NET developers
2. Propeller Mobile Native— Represents the power and driving force to transition mobile for .NET developers
3. Razor/Blazor Mobile Native — Reusing the existing prominent names from the .NET world to represent a mobile framework for .NET Developers.
4.  Brick Mobile Native - Hard as a brick.
5. Tart Mobile Native - Or food related references

**Reply by grokys** [reply]
For context: the codename for WPF was Avalon, should probably have mentioned that in the article,🤦
